backend/tests/unit/dal/Services/PropertyServiceTest.cs

```diff
@@ -0,0 +1,173 @@
+using FluentAssertions;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Moq;
+using Pims.Core.Test;
+using Pims.Dal.Entities;
+using Pims.Dal.Exceptions;
+using Pims.Dal.Security;
+using Pims.Dal.Services;
+using Xunit;
+using Microsoft.EntityFrameworkCore;
+using Pims.Core.Extensions;
+
+namespace Pims.Dal.Test.Services
+{
+    [Trait("category", "unit")]
+    [Trait("category", "dal")]
+    [Trait("group", "property")]
+    [ExcludeFromCodeCoverage]
+    public class PropertyServiceTest
+    {
+        #region Tests
+        #region GetById
+        [Fact]
+        public void GetById_Success()
+        {
+            // Arrange
+            var helper = new TestHelper();
+            var user = PrincipalHelper.CreateForPermission(Permissions.PropertyView);
+
+            var property = EntityHelper.CreateProperty(1);
```

Comment:
it does not look like the property is being used. Is it necessary to be created?

Reply:
Look at the next line - it is adding it to the in-memory DB

```cs
helper.CreatePimsContext(user, true).AddAndSaveChanges(property);
```

Reply:
We need to add it to be able to find it by ID - the whole purpose of this test

Reply:
Looks like the mock on line 36 is not checking the ID passed. Line 42 is not checking that the property passed is being returned (In this case wont be returned given the repository mocked function is not using the created context). So from that I gather that the property is not being used for the purpose of the test

Reply:
ok - will remove inserting the property